{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Product defects","children":[{"name":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > Module build failed (from ./node_modules/ts-loader/index.js):\nError: Debug Failure. False expression: Non-string value passed to `ts.resolveTypeReferenceDirective`, likely by a wrapping package working with an outdated `resolveTypeReferenceDirectives` signature. This is probably not a problem in TS itself.\n    at Object.resolveTypeReferenceDirective (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/typescript/lib/typescript.js:43500:18)\n    at /home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/ts-loader/dist/servicesHost.js:244:56\n    at /home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/ts-loader/dist/servicesHost.js:87:142\n    at Array.map (<anonymous>)\n    at Object.resolveTypeReferenceDirectives (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/ts-loader/dist/servicesHost.js:87:125)\n    at actualResolveTypeReferenceDirectiveNamesWorker (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/typescript/lib/typescript.js:119382:163)\n    at resolveTypeReferenceDirectiveNamesWorker (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/typescript/lib/typescript.js:119682:26)\n    at processTypeReferenceDirectives (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/typescript/lib/typescript.js:121185:31)\n    at findSourceFileWorker (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/typescript/lib/typescript.js:121070:21)\n    at findSourceFile (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/typescript/lib/typescript.js:120922:26)\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","children":[{"name":"Check App build and running & Check app elements exist","uid":"4645e163e4728867","parentUid":"683ff620e4fefa83ba39b6b84c0e074d","status":"failed","time":{"start":1679449923096,"stop":1679449923098,"duration":2},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"Check Main Tab Functionality","uid":"8d0b66ee8be237f3","parentUid":"683ff620e4fefa83ba39b6b84c0e074d","status":"failed","time":{"start":1679449923098,"stop":1679449923102,"duration":4},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"Check Routing elements","uid":"d7c6779cd092952b","parentUid":"683ff620e4fefa83ba39b6b84c0e074d","status":"failed","time":{"start":1679449923102,"stop":1679449923103,"duration":1},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"Check Dialog elements","uid":"8aab4f17a97159dc","parentUid":"683ff620e4fefa83ba39b6b84c0e074d","status":"failed","time":{"start":1679449923102,"stop":1679449923102,"duration":0},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"Check Svelte Page elements","uid":"23e074bc9acdcefb","parentUid":"683ff620e4fefa83ba39b6b84c0e074d","status":"failed","time":{"start":1679449923102,"stop":1679449923102,"duration":0},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"Check UI Library elements","uid":"e9e5058a1a2c4940","parentUid":"683ff620e4fefa83ba39b6b84c0e074d","status":"failed","time":{"start":1679449923102,"stop":1679449923102,"duration":0},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"683ff620e4fefa83ba39b6b84c0e074d"},{"name":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > Module build failed (from ./node_modules/ts-loader/index.js):\nError: Debug Failure. False expression: Non-string value passed to `ts.resolveTypeReferenceDirective`, likely by a wrapping package working with an outdated `resolveTypeReferenceDirectives` signature. This is probably not a problem in TS itself.\n    at Object.resolveTypeReferenceDirective (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react16/app-05/node_modules/typescript/lib/typescript.js:43500:18)\n    at /home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react16/app-05/node_modules/ts-loader/dist/servicesHost.js:244:56\n    at /home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react16/app-05/node_modules/ts-loader/dist/servicesHost.js:87:142\n    at Array.map (<anonymous>)\n    at Object.resolveTypeReferenceDirectives (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react16/app-05/node_modules/ts-loader/dist/servicesHost.js:87:125)\n    at actualResolveTypeReferenceDirectiveNamesWorker (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react16/app-05/node_modules/typescript/lib/typescript.js:119382:163)\n    at resolveTypeReferenceDirectiveNamesWorker (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react16/app-05/node_modules/typescript/lib/typescript.js:119682:26)\n    at processTypeReferenceDirectives (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react16/app-05/node_modules/typescript/lib/typescript.js:121185:31)\n    at findSourceFileWorker (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react16/app-05/node_modules/typescript/lib/typescript.js:121070:21)\n    at findSourceFile (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react16/app-05/node_modules/typescript/lib/typescript.js:120922:26)\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","children":[{"name":"Check App build and running & Check app elements exist","uid":"b5b5437073fe6100","parentUid":"7c0a94a463b85fcd5d1f4415ae037487","status":"failed","time":{"start":1679449960128,"stop":1679449960130,"duration":2},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"Check Main Tab Functionality","uid":"85d7b2db28e23c9d","parentUid":"7c0a94a463b85fcd5d1f4415ae037487","status":"failed","time":{"start":1679449960130,"stop":1679449960130,"duration":0},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"Check UI Library elements","uid":"24321f8a4cbf23a5","parentUid":"7c0a94a463b85fcd5d1f4415ae037487","status":"failed","time":{"start":1679449960130,"stop":1679449960131,"duration":1},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"7c0a94a463b85fcd5d1f4415ae037487"},{"name":"The following error originated from your test code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > window.confirm wasn't called within 1s\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nBecause this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `Check is Comprehensive Demo...`","children":[{"name":"Check Dialog elements","uid":"79d5317af627bd2d","parentUid":"008a39cd97ac7fd5a612102abe00fb7c","status":"failed","time":{"start":1679449960131,"stop":1679449960131,"duration":0},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"008a39cd97ac7fd5a612102abe00fb7c"},{"name":"Timed out retrying after 10000ms: expected '' to include 'https://reactjs.org'\n\nThe command was expected to run against origin `https://reactjs.org` but the application is at origin `https://react.dev`.\n\nThis commonly happens when you have either not navigated to the expected origin or have navigated away unexpectedly.","children":[{"name":"Checks Learn React link functionality","uid":"d756362dafa9c7d2","parentUid":"d9bc8ddc427bf68560906f0d9f4506cb","status":"failed","time":{"start":1679450483257,"stop":1679450496723,"duration":13466},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":1,"retriesStatusChange":false,"parameters":[]}],"uid":"d9bc8ddc427bf68560906f0d9f4506cb"}],"uid":"8fb3a91ba5aaf9de24cc8a92edc82b5d"},{"name":"Test defects","children":[{"name":"The following error originated from your application code, not from Cypress.\n\n  > Module build failed (from ./node_modules/ts-loader/index.js):\nError: Debug Failure. False expression: Non-string value passed to `ts.resolveTypeReferenceDirective`, likely by a wrapping package working with an outdated `resolveTypeReferenceDirectives` signature. This is probably not a problem in TS itself.\n    at Object.resolveTypeReferenceDirective (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/typescript/lib/typescript.js:43500:18)\n    at /home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/ts-loader/dist/servicesHost.js:244:56\n    at /home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/ts-loader/dist/servicesHost.js:87:142\n    at Array.map (<anonymous>)\n    at Object.resolveTypeReferenceDirectives (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/ts-loader/dist/servicesHost.js:87:125)\n    at actualResolveTypeReferenceDirectiveNamesWorker (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/typescript/lib/typescript.js:119382:163)\n    at resolveTypeReferenceDirectiveNamesWorker (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/typescript/lib/typescript.js:119682:26)\n    at processTypeReferenceDirectives (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/typescript/lib/typescript.js:121185:31)\n    at findSourceFileWorker (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/typescript/lib/typescript.js:121070:21)\n    at findSourceFile (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/typescript/lib/typescript.js:120922:26)\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","children":[{"name":"Check App build and running & Check app elements exist","uid":"5e2e46f5c19c30a8","parentUid":"0f50fdd4cb379083c974d4c8bf33ed41","status":"broken","time":{"start":1679449923602,"stop":1679449924076,"duration":474},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":true,"retriesCount":1,"retriesStatusChange":true,"parameters":[]}],"uid":"0f50fdd4cb379083c974d4c8bf33ed41"},{"name":"The following error originated from your application code, not from Cypress.\n\n  > Module build failed (from ./node_modules/ts-loader/index.js):\nError: Debug Failure. False expression: Non-string value passed to `ts.resolveTypeReferenceDirective`, likely by a wrapping package working with an outdated `resolveTypeReferenceDirectives` signature. This is probably not a problem in TS itself.\n    at Object.resolveTypeReferenceDirective (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react16/app-05/node_modules/typescript/lib/typescript.js:43500:18)\n    at /home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react16/app-05/node_modules/ts-loader/dist/servicesHost.js:244:56\n    at /home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react16/app-05/node_modules/ts-loader/dist/servicesHost.js:87:142\n    at Array.map (<anonymous>)\n    at Object.resolveTypeReferenceDirectives (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react16/app-05/node_modules/ts-loader/dist/servicesHost.js:87:125)\n    at actualResolveTypeReferenceDirectiveNamesWorker (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react16/app-05/node_modules/typescript/lib/typescript.js:119382:163)\n    at resolveTypeReferenceDirectiveNamesWorker (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react16/app-05/node_modules/typescript/lib/typescript.js:119682:26)\n    at processTypeReferenceDirectives (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react16/app-05/node_modules/typescript/lib/typescript.js:121185:31)\n    at findSourceFileWorker (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react16/app-05/node_modules/typescript/lib/typescript.js:121070:21)\n    at findSourceFile (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react16/app-05/node_modules/typescript/lib/typescript.js:120922:26)\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","children":[{"name":"Check App build and running & Check app elements exist","uid":"4eb4ea46a23cfbae","parentUid":"95021f424e4c351fe208a7e4a6ab5d0a","status":"broken","time":{"start":1679449960949,"stop":1679449961436,"duration":487},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":true,"retriesCount":1,"retriesStatusChange":true,"parameters":[]}],"uid":"95021f424e4c351fe208a7e4a6ab5d0a"}],"uid":"bdbf199525818fae7a8651db9eafe741"}],"name":"categories"}